### PR TITLE
ingress: Revise timeline for Ingress v1 GA tasks

### PR DESCRIPTION
**Pull request by cmluciano**
Signed-off-by: Christopher M. Luciano <cmluciano@us.ibm.com>

### CODE REVIEW

**Comment by cmluciano** [comment]
cc @bowei @robscott @liggitt 

I updated the sections for the proposed timeline and added a specific section for 1.18 that includes what we hope to have merged by code freeze. Thanks for the help :)

File: keps/sig-network/20190125-ingress-api-group.md [hunk 1]
```diff
@@ -638,13 +641,24 @@ backend:
   `extensions/v1beta1` until `networking.k8s.io/v1` is available.
 - [x] Update documentation to reference `networking.k8s.io/v1beta1` kubernetes/website#14239
 
-### 1.16
+### 1.18 `networking/v1beta1`
+
+- [ ] Implement [Path as a prefix](#path-as-a-prefix)
+    - [ ] New resource added with default to Implementation Specific
```
Comment (liggitt) [inline keps/sig-network/20190125-ingress-api-group.md]: new field or new resource?

File: keps/sig-network/20190125-ingress-api-group.md [hunk 2]
```diff
@@ -638,13 +641,24 @@ backend:
...
+- [ ] Implement new resource [Alternative backend types](#alternative-backend-types)
+- [ ] Implement [Hostname wildcards](#hostname-wildcards)
+- [ ] Implement new resource for [Ingress class](#ingress-class)
+    - [ ] Defaulting admission controller for [Ingress class](#ingress-class)
+
+### 1.19
 
-- [ ] Meet graduation criteria and promote API to `networking.k8s.io/v1`
 - [ ] Implement API changes to GA version.
+    - [ ] PathType is now required but has no default
```
Comment (liggitt) [inline keps/sig-network/20190125-ingress-api-group.md]: no default in v1 specifically

File: keps/sig-network/20190125-ingress-api-group.md [hunk 3]
```diff
@@ -672,12 +686,12 @@ backend:
   `networking.k8s.io`
 - [x] 1.15: all in-tree use and in-org controllers switch to
   `networking.k8s.io` API group
-- [ ] 1.15: documentation and examples are updated to refer to
+- [ ] 1.18: documentation and examples are updated to refer to
```
Comment (liggitt) [inline keps/sig-network/20190125-ingress-api-group.md]: 1.19, right?

File: keps/sig-network/20190125-ingress-api-group.md [hunk 4]
```diff
@@ -638,13 +641,24 @@ backend:
   `extensions/v1beta1` until `networking.k8s.io/v1` is available.
 - [x] Update documentation to reference `networking.k8s.io/v1beta1` kubernetes/website#14239
 
-### 1.16
+### 1.18 `networking/v1beta1`
+
+- [ ] Implement [Path as a prefix](#path-as-a-prefix)
+    - [ ] New resource added with default to Implementation Specific
+- [ ] Implement new resource [Alternative backend types](#alternative-backend-types)
```
Comment (bowei) [inline keps/sig-network/20190125-ingress-api-group.md]: should we put these as chekd off (avoid another commit)

File: keps/sig-network/20190125-ingress-api-group.md [hunk 5]
```diff
@@ -638,13 +641,24 @@ backend:
...
+- [ ] Implement [Hostname wildcards](#hostname-wildcards)
+- [ ] Implement new resource for [Ingress class](#ingress-class)
+    - [ ] Defaulting admission controller for [Ingress class](#ingress-class)
+
+### 1.19
```
Comment (sftim) [inline keps/sig-network/20190125-ingress-api-group.md]: Seems to me like “Update documentation to reference `networking.k8s.io/v1`” should be in v1.19. Does that sound right?

I'm basing this on “Announce `networking.k8s.io/v1beta1` Ingress as deprecated”.

**Comment by cmluciano** [comment]
/close 

replaced by https://github.com/kubernetes/enhancements/pull/1826

**Pull request closed without merging**